packages/docs-reanimated/docs/advanced/useEvent.mdx

```diff
@@ -4,26 +4,86 @@ sidebar_position: 4
 
 # useEvent
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+`useEvent` is a low-level hook. It returns event handler that will be called when native event occurs. You can use it to create custom event handler hooks, like `useAnimatedGestureHandler` or `useAnimatedScrollHandler`.
```

Comment:
Same here as with `useHandler` PR: `useAnimatedGestureHandler` is deprecated so I'd switch it to useScrollViewOffset or useComposedEventHandler

Reply:
And linked them